en/docs/architecture-diagram/architecture-diagram.md

```diff
@@ -69,62 +62,77 @@ isolated function getSupportedCurrencies() returns string[]|error {
 
-The execution of this command will open up a VS Code webview panel that will contain three types of design diagrams. You can navigate between these diagrams using the side panel available on the diagram webview.
+## Explore the Project Design view
 
-### Service Interaction Diagram
+Let's explore the features of the Design Diagram using the [Google Cloud Platform](https://github.com/ballerina-guides/gcp-microservices-demo/tree/project-design-demo) Ballerina project that you set up in the previous topic.
 
-The Service Interaction Diagram is the entry point of the Project Design Diagram tool. 
+When you opened the Design Diagram as explained in the previous topic, you will see three types of design diagrams to choose from as shown below. 
 
-#### Basic diagram
+![Extension Command](../img/design-diagrams.png)
 
-This diagram displays all the services available within the workspace with directed links representing any interactions between them. The Service Interaction Diagram generated for the 10 Ballerina packages in the Google Cloud Platform demo project is shown below.
+### Service Diagram: Level 1
+
+This diagram displays all the services available within the workspace with links representing any interactions between them. The basic Service Diagram (level 1) generated for the 10 Ballerina packages in the Google Cloud Platform demo project is shown below.
 
 ![Service Interaction Diagram](../img/pdd-service-interaction-diagram.png)
->**Info:** The directed link between the `Checkout` service and the `Email` service is an indication that a component of the `Checkout` service invokes the `Email` service. In addition to this, the `Email` service also invokes an `external` service via a connector.
 
-#### Detailed diagram
-The detailed Service Interaction Diagram (Level 2) dives deeper into the compositions of the services and the interactions between them. The following diagram is the detailed representation of the demo project, and it depicts the individual resource/remote functions of the services and their invocations.
+>**Info:** The link between the `Checkout` service and the `Email` service indicates that a component of the `Checkout` service invokes the `Email` service. In addition to this, the `Email` service also invokes an `external` service via a connector.
+
+### Service Diagram: Level 2
+
+The detailed Service Diagram (Level 2) dives deeper into the compositions of the services and the interactions between them. Shown below is the detailed representation of the demo project, which depicts the individual resource/remote functions of the services and their invocations.
 
 ![Detailed Service Interaction Diagram](../img/pdd-detailed-service-interaction-diagram.png)
 
 The data types of the request and response bodies of the interactions can be viewed by hovering over a particular invocation.
 
 ![Detailed Service: Request Response Data Popups](../img/pdd-request-response-data-popup.png)
 
-### Types Diagram
+### Type Diagram
```

Comment:
Any specific reason to make it `Type`? In that case need to fix that in the following text also.

Reply:
ah yes. I thought it was Type in the UI.
Let me check again

Reply:
It's Type Diagram right?

<img width="754" alt="Screenshot 2023-01-16 at 10 25 46" src="https://user-images.githubusercontent.com/21237558/212601781-236f3b2a-4cc4-482c-aa9e-b061de0ee057.png">